libtelemetry/src/main/java/com/mapbox/android/telemetry/Event.java

```diff
@@ -17,10 +17,11 @@
     VIS_GENERAL, VIS_ATTACHMENT, VIS_OBJ_DETECTION, NO_OP, CRASH
   }
 
-  static EnumSet<Type> mapGestureEventTypes = EnumSet.of(Type.MAP_CLICK, Type.MAP_DRAGEND);
-  static EnumSet<Type> navigationEventTypes = EnumSet.of(Type.NAV_DEPART, Type.NAV_ARRIVE, Type.NAV_CANCEL,
+  public static EnumSet<Type> mapGestureEventTypes = EnumSet.of(Type.MAP_CLICK, Type.MAP_DRAGEND);
```

Comment:
should these be moved altogether to `gl-native`?

Reply:
Will not use.